contract_payment_mode/__init__.py

```diff
@@ -1,2 +1,27 @@
 # -*- coding: utf-8 -*-
+# © 2016 Antiun Ingenieria S.L. - Antonio Espinosa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 from . import models
+from openerp import api, SUPERUSER_ID
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def copy_payment_mode(cr, registry):
```

Comment:
OK, this is new for me. I've copy-pasted from other OCA addon ... Good to know, thanks